src/main/target/common_pre.h

```diff
@@ -124,6 +123,10 @@
 #define USE_TIMER_MGMT
 #define USE_PERSISTENT_OBJECTS
 #define USE_DMA_RAM
+#define USE_SRAM2
```

Comment:
Not sure this does anything - I could not find any use of `SRAM2`.

Reply:
Nor USE_DMA_RAM for that matter given the change in src/main/drivers/dshot_bitbang.c. I'll update accordingly.

Reply:
No, USE_DMA_RAM is needed.